lms/djangoapps/courseware/date_summary.py

```diff
@@ -626,7 +627,7 @@ def is_enabled(self):
     @lazy
     def verification_status(self):
         """Return the verification status for this user."""
-        return SoftwareSecurePhotoVerification.user_status(self.user)[0]
+        return IDVerificationService.user_status(self.user)[0]
```

Comment:
What is the `[0]` here for?

Reply:
Ahh, I see now that it returns `(status, error_msg)`

Can we change this to avoid the magic number `0`:
```
status, ___ = IDVerificationService.user_status(self.user)
return status
```

Also would be good to update the docstring for that method call to indicate the actual return value, and note that it returns errors as part of the return value since that is odd.